game/scripts/npc/abilities/jakiro_liquid_fire.txt

```diff
@@ -49,7 +49,7 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage"                                          "12 16 20 24 48 72"
+        "damage"                                          "12 24 36 48 96 144" //OAA
```

Comment:
Buffing dmg on a spell without mana cost and 4 s cd. Seems ok.

Reply:
Your not maxing this spell first. Dual Breath is still the best spell to max. The benefit of no mana costs doesnt matter past 10 minutes. Even at level 4 48*5 = 240 damage isnt huge.